internal/controllers/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -2492,13 +2492,11 @@ func createMachinesWithNodes(
 		op(o)
 	}
 
-	var (
-		nodes         []*corev1.Node
-		machines      []*clusterv1.Machine
-		infraMachines []*unstructured.Unstructured
-	)
+	nodes := make([]*corev1.Node, 0, o.count)
```

Comment:
```suggestion
	var nodes []*corev1.Node
```

We don't always add something to nodes

Reply:
fine for me.